WooCommerce/Classes/POS/Presentation/ItemListView.swift

```diff
@@ -47,23 +61,22 @@ private extension ItemListView {
             HStack {
                 headerTextView
                 Spacer()
-                Button(action: {
-                    // TODO:
-                    // https://github.com/woocommerce/woocommerce-ios/issues/13357
-                    debugPrint("Not implemented")
-                }, label: {
-                    Image(uiImage: .infoImage)
-                })
```

Comment:
Why is this removed? There should be an info button when that initial banner is removed.
In https://github.com/woocommerce/woocommerce-ios/pull/13417/files I started updating product list and left a method `openInfoBanner()` that could call `showModal.toggle()` and we can call it from this button and when tapping the whole banner card

<img width="385" alt="Screenshot 2024-07-25 at 10 18 52" src="https://github.com/user-attachments/assets/2cb2707f-02ce-437c-a161-1b08cc4caf6d">

Reply:
I added it back. Looks like we both worked on the same code. I incorporated your styling code into mine during resolving merge conflicts.